.github/workflows/release.yml

```diff
@@ -53,8 +53,9 @@ jobs:
         include:
         - os: ubuntu-20.04
           arch: x86_64
-        - os: ubuntu-20.04
-          arch: i686
+        # Numpy no longer builds i686 packages
+        # - os: ubuntu-20.04
+        #   arch: i686
```

Comment:
cc @caspervdw just to ensure you are aware (I think you originally added  this platform to the wheel building?)

For any other project I am involved in, we also don't distribute i686 wheels, so I am certainly fine with dropping them. But just to note that theoretically we can still provide them if there is a good use case (we don't theoretically _need_ numpy wheels (practically is something else ..), and certainly for users of this platform, building numpy from source with `pip install numpy` is typically easier than shapely, as we require an external C++ dependency)

Reply:
If someone goes to the work of building numpy from source on 32-bit platforms then these should really be done as a separate project so others have access to them. That said, if numpy dropped the building of them I have a feeling it was more work than it was worth (not just build times).

That said...if someone makes 32-bit wheels for numpy let me know.

Reply:
FYI, NumPy's last built wheels for i686 were for [version 1.21.6](https://pypi.org/project/numpy/1.21.6/#files) from Apr 13, 2022. Fixing shapely to to use these older numpy's would need older cibuildwheel images to be configured to match.